src/metatrain/experimental/nativepet/default-hypers.yaml

```
@@ -37,6 +37,9 @@ architecture:
     log_separate_blocks: false
     best_model_metric: rmse_prod
     finetune: {}
+    grad_clip:
+      enable: false
+      max_grad_norm: 1.0
```

```suggestion
    grad_clip_norm: .inf
```
Adding new hypers needs to be done with care... Imagine having to navigate options with 100 different hypers. In this case, these three lines can be compressed into one as above. No gradient clipping is equivalent to gradient clipping with a norm of +inf, which is incidentally not supported by JSON but should be supported by YAML with the above syntax